can-stache-converters.js

```diff
@@ -61,8 +61,8 @@ stache.registerConverter("index-to-selected", {
 });
 
 stache.registerConverter("either-or", {
-	get: function(chosen, a, b){
-		return b !== chosen();
+	get: function(chosen, a){
+		return a === chosen();
```

Comment:
I'd suggest we even take it a step further and set `chosen(b)` if `chosen() !== a`.